<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>370</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>534</width>
    <height>370</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>534</width>
    <height>370</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Cross Impact matrix</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>500</width>
     <height>340</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>500</width>
     <height>340</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>500</width>
     <height>340</height>
    </size>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContents</enum>
   </property>
   <property name="rowCount">
    <number>8</number>
   </property>
   <property name="columnCount">
    <number>9</number>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>53</number>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>55</number>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>40</number>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>40</number>
   </attribute>
   <row>
    <property name="text">
     <string>e0</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>e1</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>e2</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>e3</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>e4</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>e5</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>e6</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>e7</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>p(e_i)</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>_</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>_</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>_</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>P(e_i,e_j)</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>_</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>_</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>_</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>_</string>
    </property>
   </column>
   <item row="0" column="0">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="0" column="1">
    <property name="text">
     <string/>
    </property>
    <property name="background">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </property>
   </item>
   <item row="1" column="2">
    <property name="background">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </property>
   </item>
   <item row="2" column="3">
    <property name="background">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </property>
   </item>
   <item row="3" column="4">
    <property name="background">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </property>
   </item>
   <item row="4" column="5">
    <property name="background">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </property>
   </item>
   <item row="5" column="6">
    <property name="background">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </property>
   </item>
   <item row="6" column="7">
    <property name="background">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </property>
   </item>
   <item row="7" column="8">
    <property name="background">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
